packages/react/stories/Typography.js

```diff
@@ -14,32 +18,38 @@ import {
   Sub1,
   Sub2
 } from "../src/hig-react";
-import arrayToObject from "../.storybook/helpers/arrayToObject";
+
+storiesOf("Typography", module).add("base component", () => (
+  <Typography
+    bold={boolean("Bold", false)}
+    color={select("Color", validColors, "hig-cool-gray-70")}
+    disabled={boolean("Disabled", false)}
+    opacity={number("Opacity", 1.0, {
+      min: 0.0,
+      max: 1.0,
+      step: 0.1
+    })}
+    size={text("Size")}
```

Comment:
FYI, I haven't seen a way for the current Select knob to provide a way to unset a prop. That seems to be in development, but for now, I've been using `text` to indicate optional `oneOf` props.

Reply:
Lets add a todo comment here, so we know to revisit it later.